beta-20161027.md

```diff
@@ -0,0 +1,81 @@
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161020.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161020.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- Functions that return the current time (including `now()`) now return values of type `TIMESTAMP WITH TIME ZONE` instead of `TIMESTAMP`, unless called from a context that expects the latter type. [#9618](https://github.com/cockroachdb/cockroach/pull/9618)
+
+### SQL Language Changes
+
+- Views are now supported.
+- Tables in the `pg_catalog` database can now be accessed with unqualified names if no table of the same name exists in the current database. [#9927](https://github.com/cockroachdb/cockroach/pull/9927)
+- `AS OF SYSTEM TIME` queries now support the decimal format returned by `cluster_logical_timestamp()` for maximum precision. [#9934](https://github.com/cockroachdb/cockroach/pull/9934)
+- The `CASCADE` option of `DROP TABLE` and `ALTER TABLE DROP COLUMN` now drops views that depend on the table or column; it is now an error to attempt to drop a table or column with dependent views without either using this option or dropping the views first. [#9724](https://github.com/cockroachdb/cockroach/pull/9724) [#10124](https://github.com/cockroachdb/cockroach/pull/10124)
+- `SET DEFAULT_TRANSACTION_ISOLATION` is now supported, improving compatibility with `psycopg2`. [#10087](https://github.com/cockroachdb/cockroach/pull/10087)
+- The `pg_catalog.pg_indexes` and `pg_catalog.pg_constraint` tables are now supported. [#9869](https://github.com/cockroachdb/cockroach/pull/9869) [#10185](https://github.com/cockroachdb/cockroach/pull/10185)
+- `EXPLAIN` can now be used with `SHOW`, `HELP`, and `ALTER TABLE ... SPLIT`. [#10013](https://github.com/cockroachdb/cockroach/pull/10013)
+- The `current_schemas()` function is now supported, improving compatibility with ActiveRecord. [#9604](https://github.com/cockroachdb/cockroach/pull/9604)
+- The `SHOW ALL` command is now supported. [#10195](https://github.com/cockroachdb/cockroach/pull/10195)
+- The `DROP INDEX` and `ALTER INDEX` commands can now use unqualified index names. [#10091](https://github.com/cockroachdb/cockroach/pull/10091)
+
+### Command-Line Interface Changes
+
+- The `--host` argument now defaults to `localhost` in both secure and insecure modes. [#10134](https://github.com/cockroachdb/cockroach/pull/10134)
+- Improved error messages when a server is down or certificates are not configured correctly. [#9917](https://github.com/cockroachdb/cockroach/pull/9917)
+- The `--key` argument can now be passed via the environment variable `COCKROACH_KEY`, matching other similar flags. [#10092](https://github.com/cockroachdb/cockroach/pull/10092)
+
+### Performance Improvements
```

Comment:
We haven't been documenting any other features that are enabled by environment variables. I think it's just going to confuse things to say that this feature (which is subtle to begin with) exists but you shouldn't use it.